cli/src/print.rs

```diff
@@ -11,26 +11,26 @@ use crate::summarize::Summarize;
 
 #[macro_export]
 macro_rules! print_user_success {
-    ($($tts:tt)*) => {
+    ($($tts:tt)*) => {{
         eprint!("✅ ");
         eprintln!($($tts)*);
-    }
+    }}
 }
 
 #[macro_export]
 macro_rules! print_user_warning {
-    ($($tts:tt)*) => {
+    ($($tts:tt)*) => {{
         eprint!("⚠️  ");
         eprintln!($($tts)*);
-    }
+    }}
 }
 
 #[macro_export]
 macro_rules! print_user_failure {
-    ($($tts:tt)*) => {
+    ($($tts:tt)*) => {{
         eprint!("❗ ");
         eprintln!($($tts)*);
-    }
+    }}
```

Comment:
These macros omit the double brace syntax required for using them in things like match arms without trailing semicolon. It's usually good practice to allow that to make macros feel more like native Rust syntax. So I've added them for everything here.

Reply:
Good catch.